docs/openapi/api-external-b2b-pa-v1.yaml

```diff
@@ -207,7 +207,17 @@ info:
     </details></br>
 
     ><details>
-      <summary><strong>Cos'è il codice di tassonomia(taxonomyCode) ?</strong></summary>
+      <summary><strong>Quali controlli vengono effettuati in seguito all'inserimento della notifica?</strong></summary>
+      <em>I controlli effettuati da PN sono i seguenti:</em><ul><li><em>verifica che le informazioni inserite siano sintatticamente corrette</em></li><li><em>verifica che le combinazioni di <strong>creditorTaxId/noticeCode</strong> e <strong>creditorTaxId/alternativeNoticeCode</strong> siano univoche</em></li><li><em>verifica che la combinazione dei campi <strong>senderPaId, paProtocolNumber, idempotenceToken</strong> siano univoci</em></li></ul>
+    </details></br>
+    
+    ><details>
+      <summary><strong> Cosa rappresentano i campi requiresAck e sendByMail? <font color="red">queste funzioni sono in fase di sviluppo</font> </strong></summary>
```

Comment:
Più che in fase di sviluppo, metterei "non ancora implementate"

Reply:
modificato titolo il "queste funzioni non sono ancora implementate"